app/_src/gateway/reference/configuration.md

```diff
@@ -4281,6 +4280,6 @@ Notes:
 **Default:** none
 
 
-[Penlight]: http://stevedonovan.github.io/Penlight/api/index.html
-[pl.template]: http://stevedonovan.github.io/Penlight/api/libraries/pl.template.html
-[templates]: https://github.com/kong/kong/tree/master/kong/templates
+[Penlight](http://stevedonovan.github.io/Penlight/api/index.html)
+[pl.template](http://stevedonovan.github.io/Penlight/api/libraries/pl.template.html)
+[templates](https://github.com/kong/kong/tree/master/kong/templates)
```

Comment:
@acgoldsmith I think these ones are a different type of link, don't know what the name is, but basically they don't appear in the output docs, but they reference a link, so all you have to do is something like "For more info, see [Penlight](Penlight)." for example and it will pull in the right link. So you can kind of reuse links on a page and won't have to keep defining them. So I think this one was correct the way it originally was.